pkg/handler/crud.go

```diff
@@ -68,6 +70,34 @@ var (
 func (c *crud) FindFlags(params flag.FindFlagsParams) middleware.Responder {
 	fs := []entity.Flag{}
 	q := entity.NewFlagQuerySet(getDB())
+	queryParams := map[string]string{
```

Comment:
I think it's better to put the query params in swagger query params.

https://github.com/checkr/flagr/blob/master/swagger/flags.yaml#L1

For example, https://swagger.io/docs/specification/describing-parameters/#query-parameters

After the change of the swagger definitions, run `make gen` to get the latest swagger model. We can get type safety and API doc for free in this case.